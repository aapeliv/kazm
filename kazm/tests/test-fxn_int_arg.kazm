//fxn with int arg called in main 

void func(int val) {
  int_println(val);
}

int main() {
  println("About to call func(5)");
  func(5);
  println("Done.");
}
