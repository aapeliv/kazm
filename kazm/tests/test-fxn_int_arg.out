About to call func(5)
5
Done.
